src/cpp/src/tokenizer.cpp

```diff
@@ -206,6 +249,9 @@ class Tokenizer::TokenizerImpl {
         if (ov_tokenizer) {
             ov::pass::Manager manager;
             manager.register_pass<MakeCombineSegmentsSatateful>();
+            manager.register_pass<MakePaddingSatateful>();
+            manager.register_pass<MakeTruncationSatateful>();
+            manager.register_pass<ov::pass::VisualizeTree>("after.svg");
```

Comment:
it's used only for debug, please, remove when code is ready

Reply:
sorry🤦will remove that

Reply:
removed